webHandler.go

```diff
@@ -122,6 +122,8 @@ func (m WebHanlder) respond(data response.RespData, abort bool, w webFramework.W
 	} else {
 		resp.ErrorData = m.GetErrorsArray(data.Message, data)
 
+		w.Parser.SetLocal("downTime", data.Message)
```

Comment:
hi, please use better local name instead of `downTime`, and use `res.ErrorData` instead of `data.Message`